activitysim/abm/models/trip_destination.py

```diff
@@ -947,6 +950,18 @@ def sample_skims(self, presample):
             "pdt_skims": skim_dict.wrap_3d(
                 orig_key=p, dest_key=d, dim3_key="trip_period"
             ),
+            "opt_skims": skim_dict.wrap_3d(
+                orig_key=o, dest_key=p, dim3_key="trip_period"
+            ),
+            "pot_skims": skim_dict.wrap_3d(
+                orig_key=p, dest_key=o, dim3_key="trip_period"
+            ),
+            "ndt_skims": skim_dict.wrap_3d(
+                orig_key=n, dest_key=d, dim3_key="trip_period"
+            ),
+            "dnt_skims": skim_dict.wrap_3d(
+                orig_key=d, dest_key=n, dim3_key="trip_period"
+            ),
```

Comment:
This looks probably fine for "legacy" use, but it's going to break the sharrow integration.  There's a parallel data structure for sharrow defined [here](https://github.com/camsys/activitysim/blob/3770e5af5517b311ee40bf9dd381212f0c979f69/activitysim/core/flow.py#L323) that will need to be updated if this will work with sharrow.

Reply:
Confirmed this breaks the sharrow integration. 
`14/12/2022 11:34:11 - ERROR - activitysim.core.flow - error in apply_flow: name 'op_skims' is not defined`

```python
  File "c:\users\wangs1\workspace\activitysim\activitysim\core\interaction_sample_simulate.py", line 161, in _interaction_sample_simulate
    ) = interaction_simulate.eval_interaction_utilities(
  File "c:\users\wangs1\workspace\activitysim\activitysim\core\interaction_simulate.py", line 203, in eval_interaction_utilities
    sh_util, sh_flow = apply_flow(
  File "c:\users\wangs1\workspace\activitysim\activitysim\core\flow.py", line 774, in apply_flow
    flow_result = flow.dot(
  File "c:\users\wangs1\workspace\sharrow\sharrow\flows.py", line 1784, in dot
    return self._load(
  File "c:\users\wangs1\workspace\sharrow\sharrow\flows.py", line 1562, in _load
    result = self.iload_raw(source, runner=runner, dtype=dtype, dot=dot)
  File "c:\users\wangs1\workspace\sharrow\sharrow\flows.py", line 1476, in iload_raw
    raise NameError(problem.group(1)) from err
NameError: name 'op_skims' is not defined
```